package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Make your Backbone.js apps dance!",
   "version": "2.4.1",
   "homepage": "https://github.com/marionettejs/backbone.marionette",
-  "main": "lib/core/backbone.marionette.js",
+  "main": "dist/backbone.marionette.js",
```

Comment:
we may want to keep the dist locations the same as before?  particularly if we're keeping the core build?  @jasonLaster